modules/terraform-aci-bridge-domain/variables.tf

```diff
@@ -98,6 +98,12 @@ variable "clear_remote_mac_entries" {
   default     = false
 }
 
+variable "multicast_arp_drop" {
+  description = "Drop ARP with Multicast SMAC."
+  type        = bool
+  default     = false
```

Comment:
please test this with null as default since the attribute is not supported in earlier versions, we cannot push false/"no" in this case